### DISCUSSION TITLE
Multiple tenant with the same account

### DISCUSSION THREAD

**Discussion by Filisteos** [Micke-K/IntuneManagement#213] [Q&A] [answered]
Hi @Micke-K, a while ago I made the suggestion that it would be great to be able to access multiple tenants from the same Microsoft account, because it facilitates management and avoids having accounts with local privileges for each tenant, which is very problematic at the management level. and security. Today I read the documentation again and I was surprised to see that you have implemented it!!

"The scripts can export and import objects including assignments and support import/export between tenants"
"Enable Get Tenant List in Settings if accessing multiple environments with the same account e.g. a guest account in other tenants. This might cause a Consent prompt"

The problem is that I can't see how it works, I don't see a button to switch the tenant, could you help me? On the other hand, I see that you have also implemented import/export between tenants, is that correct? Thanks again!!

**Comment by Micke-K** [comment, marked as answer]
Hello,

Multiple tenants have been available for a long time.

The 'Enable Get Tenant List' will get all the tenants your account has access to eg tenants you gave been invited to. This will query an azure API so it must have access to it and may cause a consent prompt. The list will be available in the profile info eg click on your profile in the upper right corner after login. Check the log if you have enabled the setting, you account is a guest in one or more tenants and these tenants doesn't show up in the list. Note that a restart is required after you enable it. 

You can also select to login with a different account from there. You can the easily swap between tenants via the profile info. 

Cheers!

**Reply by Filisteos** [reply]
Hi! After activating that option I had to log out of all accounts, close the app and add my account again, it asked me for special consent, from there the entire list of the rest of the tenants to whom I am invited appeared. Thank you!!

**Reply by Filisteos** [reply]
Micke, is it possible to send a specific policy from one tenant to another tenant, without the need to previously export? That is, a more direct way to dump content from one tenant to another.

**Reply by Micke-K** [reply]
Hello!

Not possible at the moment. I have it in my plans though. It's just taking waaaaay longer than I thought. 

Cheers!

**Reply by Filisteos** [reply]
You're the best! Thanks for everything!

**Comment by Gr0p3r** [comment]
Similarly, I would like to have multi-tenant access. I have enabled it, but it does not chnage anything. I have not been invited to any extra tenants, I am the service provider and and GDAP roles granting access as a service provider. Any way to make that work?

**Comment by Micke-K** [comment]
Hello,

The 'Enable Get Tenant List' uses an API to get the list of tenants the user has access to. Either home tenant or where invited as guest.

GDAP is not supported. It uses different APIs. I don't have service provider access to customers so I can't develop and test it.

Cheers!
